src/components/IconLayer.tsx

```diff
@@ -14,13 +15,13 @@ type IconLayerProps = {
 const IconLayer: React.FC<IconLayerProps> = (props) => {
   const iconProps = { style: props.style };
   switch(props.type) {
-    case "fill-extrusion": return <BsDiamondFill {...iconProps} />;
-    case "raster": return <BsDiamond {...iconProps} />;
+    case "fill-extrusion": return <MdOutlineVerticalShades {...iconProps} />;
+    case "raster": return <BsDiamondFill {...iconProps} />;
     case "hillshade": return <BsSun {...iconProps} />;
-    case "color-relief": return <MdBubbleChart {...iconProps} />;
-    case "heatmap": return <BsDiamond {...iconProps} />;
-    case "fill": return <BsDiamond {...iconProps} />;
-    case "background": return <BsDiamondFill {...iconProps} />;
+    case "color-relief": return <CiMountain1 {...iconProps} />;
+    case "heatmap": return <MdBubbleChart {...iconProps} />;
+    case "fill": return <BsDiamondFill {...iconProps} />;
+    case "background": return <MdOutlineStreetview {...iconProps} />;
     case "line": return <IoAnalyticsOutline {...iconProps} />;
     case "symbol": return <BsFonts {...iconProps} />;
```

Comment:
🤔 symbol also still gets the font icon, lets change this too.

This is tough.

What about

<img width="134" height="108" alt="image" src="https://github.com/user-attachments/assets/e564e599-ba6c-4ad4-9988-7307c36e7a21" />
<img width="109" height="93" alt="image" src="https://github.com/user-attachments/assets/e8bfb164-fed7-4356-8c29-b59a87a9227d" />
<img width="134" height="117" alt="image" src="https://github.com/user-attachments/assets/6dd6d0f3-41cc-4b79-ba4e-55dc15fa613b" />

Reply:
I'm not sure it the first two will look good since it is small. Location icon is probably as bad as font I guess...
Feel free to use any of the two...